### PR TITLE
Style consumer index

### PR DESCRIPTION
**Pull request by thisisgoldman**

### CODE REVIEW

**Comment by thisisgoldman** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/152?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#152** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/152?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#150** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/150?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @thisisgoldman and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: assets/svelte/consumers/Index.svelte [hunk 1]
```diff
@@ -109,8 +117,39 @@
...
+                class:bg-green-50={consumer.type === "pull"}
+                class:text-green-400={consumer.type === "pull"}
+                class:bg-blue-50={consumer.type === "push"}
+                class:text-blue-400={consumer.type === "push"}
+              >
+                {#if consumer.type === "pull"}
+                  <ArrowLeftFromLine class="h-4 w-4 mr-1 text-green-400" />
+                {:else if consumer.type === "push"}
+                  <ArrowRightToLine class="h-4 w-4 mr-1 text-blue-400" />
+                {/if}
```
Comment (RTLS) [inline assets/svelte/consumers/Index.svelte:134]: switch

so this blows up when we inevitably forget to add a case later

File: assets/svelte/consumers/Index.svelte [hunk 2]
```diff
@@ -109,8 +117,39 @@
...
+            </div>
+          </Table.Cell>
+          <Table.Cell>
+            <span
+              class="inline-flex items-center px-2.5 py-0.5 rounded-full text-xs font-medium"
+              class:text-purple-500={consumer.status === "active"}
+              class:text-gray-400={consumer.status === "disabled"}
+            >
+              {#if consumer.status === "active"}
+                <CirclePlay class="h-4 w-4 mr-1" />
```
Comment (RTLS) [inline assets/svelte/consumers/Index.svelte:146]: ditto

**Review by RTLS — APPROVED** [review]

File: assets/svelte/consumers/Index.svelte (hunk 1 above)
Comment (thisisgoldman) [reply assets/svelte/consumers/Index.svelte:134]: Svelt throws a warning when I move it to a switch. I'll keep as an if-else for now

**Comment by thisisgoldman** [comment]
### Merge activity

* **Aug 26, 8:19 PM EDT**: @thisisgoldman started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/sequinstream/sequin/152).

**Pull request merged**